library/kani_macros/src/sysroot/contracts/replace.rs

```diff
@@ -70,6 +70,7 @@ impl<'a> ContractConditionsHandler<'a> {
     /// `use_nondet_result` will only be true if this is the first time we are
     /// generating a replace function.
     fn expand_replace_body(&self, before: &[Stmt], after: &[Stmt]) -> TokenStream {
+        let redefs = &self.redefs;
```

Comment:
Instead of doing it here, you can create the redefinition inside the initialize function here: https://github.com/model-checking/kani/blob/579151659252603b894e0af8ba55878cd4cd1947/library/kani_macros/src/sysroot/contracts/replace.rs#L22